spec/laa_crime_schemas/structs/crime_application_spec.rb

```diff
@@ -76,6 +76,10 @@
           expect(subject.pre_cifc_reference_number).to eq 'pre_cifc_maat_id'
           expect(subject.pre_cifc_maat_id).to eq '987654321'
         end
+
+        it 'does not have any interests_of_justice' do
+          expect(subject.interests_of_justice.size).to eq 0
+        end
```

Comment:
Hm is this correct for a crime application?

Reply:
Ah i didn't look at the describe block, why just test the interests of justice attribute and not the rest though e.g. co defendants, offences etc..

Reply:
Oops ignore me!